assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -193,11 +140,27 @@ export const TelemetryOptInBanner = forwardRef(
             </TranslateWithMarkup>
           </LabelText>
         </Label>
-        <VisitSettingsText>
-          {__(
-            'You can update your selection later by visiting Settings.',
-            'web-stories'
-          )}
+        <VisitSettingsText
+          as="span"
+          size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+        >
+          <TranslateWithMarkup
+            mapping={{
+              a: (
+                <NavLink
+                  as="a"
+                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+                  href={APP_ROUTES.EDITOR_SETTINGS}
+                  aria-label={__('Settings', 'web-stories')}
+                />
```

Comment:
this is taking me to `http://localhost:8899/editor-settings` and not `http://localhost:8899/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/editor-settings`

I think the route has to be wrapped in our `resolveRoute` fn from `dashboard/app/router/route.js` so that it plays nicely with our hash router in here

Reply:
nice catch!